spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
+  let(:user){FactoryBot.create :user, name: "Name 2", email: "test123456789@mail.com", role: "suppervisor"}
+  let(:user1){FactoryBot.create :user, name: "Name 2", email: "test12345678910@mail.com", role: "trainee"}
+
+
+  before{login_as user}
+
+  describe "GET /index" do
```

Comment:
phân biệt giú anh describe, context và it đi

Reply:
**Describe** 
- dùng để nhóm các test case theo class, module, method hoặc theo action của controller
**Context**
- dùng để nhóm các điều kiện của test case
**It**
- được dùng để định nghĩa một test case